packages/wallets/src/features/cashier/modules/WithdrawalCrypto/components/WithdrawalCryptoReceipt/WithdrawalCryptoReceipt.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { WalletButton, WalletCard, WalletText } from '../../../../../../components';
+import ArrowDown from '../../../../../../public/images/ic-back-arrow.svg';
+import { WithdrawalCryptoDestinationAddress } from './components';
+import './WithdrawalCryptoReceipt.scss';
+
+type TProps = {
+    onClose: () => void;
+    withdrawalReceipt: {
+        address: string;
+        amount: string;
+        currency?: string;
+    };
+};
+
+const WithdrawalCryptoReceipt: React.FC<TProps> = ({ onClose, withdrawalReceipt }) => {
+    const history = useHistory();
+    const { address, amount, currency } = withdrawalReceipt;
+
+    return (
+        <div className='wallets-withdrawal-crypto-receipt'>
+            <div className='wallets-withdrawal-crypto-receipt__accounts-info'>
+                <WalletCard
+                    balance={`-${amount} ${currency}`}
+                    currency={currency ?? ''}
+                    iconSize='md'
+                    landingCompanyName='SVG'
```

Comment:
understand that user can only do crypto withdrawals using only SVG accs, but i would like to suggest you to pass landing company as a prop from the parent, considering that we have activeWallet in the parent